.storybook/preview-head.html

```diff
@@ -24,4 +24,7 @@
     font-family: var(--font-family-monospace) !important;
     font-size: var(--font-size-code) !important;
   }
+  .docblock-colorpalette div:not(:first-child) {
+    grid-column-gap: 16px;
+  }
 </style>
```

Comment:
Adds gap to `<ColorPaletteItem /> storybook docs component. Admittedly I'm probably not using it as intended and trying to squeeze the description into the subtitle but I think it works for now.